activerecord/test/cases/migration/columns_test.rb

```diff
@@ -142,6 +142,10 @@ def test_remove_column_with_index
       end
 
       def test_remove_column_with_multi_column_index
+        # MariaDB starting with 10.2.8
+        # Dropping a column that is part of a multi-column UNIQUE constraint is not permitted.
+        skip if current_adapter?(:Mysql2Adapter) && @connection.mariadb? && @connection.version >= "10.2.8"
```

Comment:
`@connection` is only used in this line in the file.
I prefer to use `connection` rather than `@connection`.

Reply:
Understood. I have another question. 
I'm wondering `if current_adapter?(:Mysql2Adapter) && connection.mariadb?` is redundant
since `mariadb?` is only available in mysql2 adapter. I have no strong preference but wanted to hear from your opinion.

Reply:
I agree with you, but we have no `connection.mariadb?` in test cases for now, so I'm acceptable that.

```
% git grep -n -i 'mariadb' test
test/cases/adapters/mysql2/datetime_precision_quoting_test.rb:22:  test "microsecond precision for MariaDB gte 5.3.0" do
test/cases/adapters/mysql2/datetime_precision_quoting_test.rb:23:    stub_version "5.5.5-10.1.8-MariaDB-log" do
test/cases/adapters/mysql2/datetime_precision_quoting_test.rb:28:  test "no microsecond precision for MariaDB lt 5.3.0" do
test/cases/adapters/mysql2/datetime_precision_quoting_test.rb:29:    stub_version "5.2.9-MariaDB" do
```

Reply:
Thanks for the reply. Modified to use `connection` here and keeping `if current_adapter?(:Mysql2Adapter) && connection.mariadb?` condition.